### PR TITLE
Add service `vacuum_reset_consumable` for xiaomi_miio

### PR DESCRIPTION
**Pull request by zry98**
## Proposed change

Add service call `vacuum_reset_consumable` to reset consumable status (e.g., main brush work time) of Xiaomi vacuum cleaners, no longer need to use MiHome app to do that.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/91483
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by azogue** [comment]
@zry98, it looks like one unrelated commit to source/_integrations/zwave_js.markdown got mixed in this PR when redirecting to 'next' instead of 'current', please clean the branch 👍

**Comment by zry98** [comment]
> @zry98, it looks like one unrelated commit to source/_integrations/zwave_js.markdown got mixed in this PR when redirecting to 'next' instead of 'current', please clean the branch 👍

I think I've messed up the branch, I shouldn't have used the `current` branch for docs of new features, right?

File: source/_integrations/xiaomi_miio.markdown [hunk 1]
```diff
@@ -1635,6 +1636,22 @@ automation:
           segments: [1, 1]
 ```
 
+### Service `xiaomi_miio.vacuum_reset_consumable`
```
Comment (frenck) [inline source/_integrations/xiaomi_miio.markdown:1639]: As per comment on the linked Core PR. These should become buttons, not services.

File: source/_integrations/zwave_js.markdown [hunk 2]
```diff
@@ -859,3 +859,16 @@ Z-Wave JS writes details to its logs. To access these logs go to the following.
    **Settings** -> **Devices & Services** -> **Integrations(tab)** -> **Z-Wave (CONFIGURE)** -> **Logs(tab)**
 
 You need to keep this browser tab open for logging to be active.
+
+## Z-Wave terminology
```
Comment (frenck) [inline source/_integrations/zwave_js.markdown:863]: Unrelated

**Review by frenck — CHANGES_REQUESTED** [review]
It seems like your branch was based on `current` and not the `next` branch. Please git rebase your changes onto the next branch to clean it up.

../Frenck

**Comment by zry98** [comment]
Messed up branches, closing it to open a new one with `next` branch.

**Pull request closed without merging**